### ISSUE TITLE
[ehancement] Optionally log in chat calls and duration

### ISSUE THREAD

**Issue by Tha14** [evilcorpltd/aTox#752]
**This is for way after 0.7.0 if it's within scope!!!**
Would be nice to have an option to enable logging calls in the chat(in the middle) and show duration and date.
![calls-log-mock](https://user-images.githubusercontent.com/5931740/120025202-036a6b80-bff9-11eb-8c64-b0daf9785ba2.jpg)

**Comment by robinlinden** [comment]
Yeah, this is on my todo for 0.7.0, but I guess I've forgotten to put it on GitHub. The same "notice" mechanism will probably be used to log finished file transfers. Thank you for the request!
